provider/app.py

```diff
@@ -31,27 +31,36 @@
 def postTrigger(namespace, trigger):
     body = request.get_json(force=True, silent=True)
     triggerFQN = '/' + namespace + '/' + trigger
-    logging.info("[{}] Ensuring user has access rights to post a trigger".format(triggerFQN))
-
-    trigger_get_response = requests.get(body["triggerURL"])
-    trigger_get_status_code = trigger_get_response.status_code
-    logging.info("[{}] Repsonse status code from trigger authorization {}".format(triggerFQN, trigger_get_status_code))
-
-    if trigger_get_status_code == 200:
-        logging.info("[{}] User authenticated. About to create consumer {}".format(triggerFQN, str(body)))
-        createAndRunConsumer(triggerFQN, body)
-        logging.info("[{}] Finished creating consumer.".format(triggerFQN))
-        result = {'success': True}
-    elif trigger_get_status_code == 401:
-        logging.info("[{}] User not authorized to post trigger".format(triggerFQN))
+
+    if triggerFQN in consumers:
+        logging.info("[{}] Trigger already exists".format(triggerFQN))
+        trigger_get_status_code = 403
```

Comment:
@bjustin-ibm, why do you think 400 is better to use than the unauthorized error code?